examples/opengl/OpenGL.cpp

```diff
@@ -58,10 +58,7 @@ int main()
         window.setMaximumSize(sf::Vector2u(1200, 900));
 
         // Create a sprite for the background
-        sf::Texture backgroundTexture;
-        backgroundTexture.setSrgb(sRgb);
-        if (!backgroundTexture.loadFromFile(resourcesDir() / "background.jpg"))
-            return EXIT_FAILURE;
+        const auto       backgroundTexture = sf::Texture::loadFromFile(resourcesDir() / "background.jpg", sRgb).value();
```

Comment:
Not going to repeat myself for all the other ones, but all of these `.value()` calls in the examples change behavior and such changes were reverted in #3051 IIRC.

Maybe we should have a policy about throwing in examples?